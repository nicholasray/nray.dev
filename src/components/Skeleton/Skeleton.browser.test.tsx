import { render } from "vitest-browser-react";
import { Skeleton } from "./Skeleton";

it("renders correctly", async () => {
  const screen = render(<Skeleton data-testid="skeleton" />);

  await expect.element(screen.getByTestId("skeleton")).toBeInTheDocument();

  expect(screen.container).toMatchSnapshot();
});
