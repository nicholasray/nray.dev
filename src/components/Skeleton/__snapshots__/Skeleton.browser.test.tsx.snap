// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<div>
  <div
    class="animate-pulse rounded-md bg-primary/10"
    data-testid="skeleton"
  />
</div>
`;
